src/EditorFeatures/VisualBasicTest/GenerateDefaultConstructors/GenerateDefaultConstructorsTests.vb

```diff
@@ -27,8 +28,7 @@ Imports System.Collections.Generic
 Imports System.Linq
 Class Program
     Inherits Exception
-    Public Sub New(message As String)
-        MyBase.New(message)
+    Public Sub New()
```

Comment:
these changes are expected given that this feature now will offer to generate the default constructor if the base type has one as well.